src/quadratic.jl

```diff
@@ -0,0 +1,66 @@
+"""
+Helper function for quadratic_interval
+"""
+function quadratic_helper(a::Number, b::Number, c::Number, L::AbstractArray)
+
+    Δ = b*b - 4*a*c
+
+    Δ < 0 && return
+
+    Δ = sqrt(Δ)
+
+    if (b >= 0)
+        x0 = -0.5 * (b + Δ)
```

Comment:
Do the calculations in this function not need directed rounding?

Reply:
They can be `Float`s, so I think interval arithmetic can be used to bound rounding errors. Should I update to include that?